apps/e2e/test/e2e_test.exs

```diff
@@ -215,6 +215,16 @@ defmodule E2ETest do
         Enum.each(expected, fn app -> assert [ds.id, app] in actual end)
       end)
     end
+
+    test "events have been stored in estuary" do
+      table = Application.get_env(:estuary, :table_name)
+
+      eventually(fn ->
+        [[actual]] = query("SELECT count(1) FROM #{table}")
+
+        assert actual > 0
```

Comment:
Do we have any evidence that the count wasn't greater than 0 before this test?

Reply:
Yup. I'll come over.

Reply:
We re-ran the test without calling data writer and our test timed out/failed because it found 0 events inserted.